app/controllers/ops_controller/settings/common.rb

```diff
@@ -835,43 +835,43 @@ def settings_get_form_vars
 
       w = qwb[:generic_worker]
       w[:count] = params[:generic_worker_count].to_i if params[:generic_worker_count]
-      w[:memory_threshold] = params[:generic_worker_threshold] if params[:generic_worker_threshold]
+      w[:memory_threshold] = params[:generic_worker_threshold].to_i if params[:generic_worker_threshold]
```

Comment:
Shouldn't we be using `to_i_with_method` here?

Or is every single use of `200.megabytes` gone?

Because...

200.megabytes.to_i == 200.megabytes == 209715200

but

"200.megabytes".to_i == 200

whereas 

"200.megabytes".to_i_with_method == 200.megabytes.to_i_with_method == 209715200

Reply:
Cc @jrafanie ^

Reply:
From log:
```
Processing by OpsController#settings_form_field_changed as JS
INFO -- :   Parameters: {"generic_worker_threshold"=>"367001600", "id"=>"workers"}
```

i.e. what you're getting from the browser here is `"367001600"`, not "350.megabytes".

Reply:
Aaaah, never mind :)

It looked like the data could come from the db, didn't notice it's always `params`.

You're right :+1:

Reply:
Thanks for digging into this.  Yes, either `"350.megabytes"`(string) or `367001600`(integer) would be acceptable, but not `"367001600"`(string).  This change looks good to me although I'm not quite sure of how the values are transformed from form to backend.